functions/Set-DbaRecoveryModel.ps1

```diff
@@ -0,0 +1,126 @@
+
+            Website: https://dbatools.io
+            Copyright: (C) Chrissy LeMaire, clemaire@gmail.com
+            License: GNU GPL v3 https://opensource.org/licenses/GPL-3.0
+
+        .LINK
+            https://dbatools.io/Set-DbaRecoveryModel
+
+        .EXAMPLE
+            Set-DbaRecoveryModel -SqlInstance sql2014 -RecoveryModel Full -AllDatabases -Verbose
+            
+            Sets the Recovery Model to Full for all user databases on SQL Server instance sql2014
+            
+        .EXAMPLE
+            Set-DbaRecoveryModel -SqlInstance sql2014 -RecoveryModel BulkLogged -Database TestDB
+            
+            Sets the Recovery Model to BulkLogged for database [TestDB] on SQL Server instance sql2014
+            
+        .EXAMPLE
+            Set-DbaRecoveryModel -SqlInstance sql2014 -RecoveryModel Simple -Database TestDB1, TestDB2 -Verbose
+            
+            Sets the Recovery Model to Simple for databases [TestDB1] and [TestDB2] on SQL Server instance sql2014
+    #>
+    [CmdletBinding()]
+    param (
+        [parameter(Mandatory = $true, ValueFromPipeline = $true)]
+        [Alias("ServerInstance", "SqlServer")]
+        [DbaInstanceParameter[]]$SqlInstance,
+        [PSCredential][System.Management.Automation.CredentialAttribute()]
+        $SqlCredential,
+        [ValidateSet('Simple', 'Full', 'BulkLogged')]
+        [string]$RecoveryModel = '',
+        [Alias("Databases")]
+        [object[]]$Database,
+        [object[]]$ExcludeDatabase,
+        [switch]$AllDatabases,
+        [parameter(ValueFromPipeline = $true)]
+        [switch][Alias('Silent')]$EnableException
+    )
+    process {
+        foreach ($serverName in $SqlInstance) {
+            try {
+                $server = Connect-SqlInstance -SqlInstance $serverName -SqlCredential $SqlCredential -MinimumVersion 11
+            }
+            catch {
+                Stop-Function -Message "Failure" -Category ConnectionError -ErrorRecord $_ -Target $instance -Continue
+            }
+            
+            $dbs = @()
+            if (!$Database -and !$AllDatabases -and !$ExcludeDatabase) {
+                Stop-Function -Message "You must specify -AllDatabases or -Database to continue"
+                return
+            }
+    
+            $dbs = Get-DbaDatabase -ServerInstance $serverName | Where-Object {$_.IsSystemObject -ne $true}
+            
+            # filter collection based on -Databases/-Exclude parameters
+            if ($Database) {
+                $dbs = $dbs | Where-Object { $Database -contains $_.Name }
+            }
+
+            if ($ExcludeDatabase) {
+                $dbs = $dbs | Where-Object { $ExcludeDatabase -notcontains $_.Name }
+            }
+            
+            if (!$dbs) {
+                Stop-Function -Message "The database you specified does not exist on the server $serverName"
+                return
+            }
+
+            foreach ($db in $dbs) {
```

Comment:
Changing `$dbs` to `$databases` it is ok to leave the single object here as just `$db`.